frontend/sw360-portlet/src/main/webapp/html/licenses/includes/editDetailSummary.jspf

```diff
@@ -50,21 +51,23 @@
   <tr>
     <td width="33%">
       <label class="checkboxlabel" for="gpl2">
-        <input id="gpl2" type="checkbox"
-               name="<portlet:namespace/><%=License._Fields.GPLV2_COMPAT%>"
-               value="<sw360:out value="${licenseDetail.GPLv2Compat}"/>"
-                <core_rt:if test="${licenseDetail.GPLv2Compat == 'true'}"> checked="checked" </core_rt:if>
-        />
-        GPL-2.0 compatibility</label>
+        GPL-2.0 compatibility:
+         <select class="toplabelledInput" id="gpl2"
```

Comment:
`EditTernary` only replaces the inner part of the select. I have removed it since it was incomplete.